Package.swift

```diff
@@ -1,9 +1,15 @@
 import PackageDescription
 
 let package = Package(
-  name: "SnowMan",
-  dependencies: [
-    .Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,7))
-  ]
+  	name: "SnowMan",
+  	targets: [
+  		Target(name: "Dictionary"),
+  		Target(name: "SnowManLogic", dependencies: ["Dictionary"]),
```

Comment:
I left it in because I was getting this:
```
Undefined symbols for architecture x86_64:
  "__TFC10Dictionary17DictionaryManagerCfT_S0_", referenced from:
      _main in main.swift.o
  "__TMaC10Dictionary17DictionaryManager", referenced from:
      _main in main.swift.o
ld: symbol(s) not found for architecture x86_64```